types/miniprogram/wx/lib.wx.app.d.ts

```diff
@@ -0,0 +1,258 @@
+/*! *****************************************************************************
```

Comment:
@sandersn do we allow copyrights in the packages here?

Reply:
I don't think it's a good idea, but since it's the MIT license (same as DT itself) it's not a problem.